src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -410,6 +473,10 @@ impl Container {
             eval_ inputs.enable_fullscreen (model.enable_fullscreen());
             eval_ inputs.enable_fullscreen (fullscreen.set_target_value(1.0));
             eval  inputs.set_size          ((s) size.set_target_value(*s));
+            inputs.clicked <+ model.view.overlay.events.mouse_down;
+            eval_ inputs.clicked([] {
+                Container::set_all_visualizations_pointer_events("auto");
+            });
```

Comment:
The frp here is done badly, and have unused endpoints, therefore:
* Remove on_clicked, clicked, clicked_outside
* then apply
```suggestion
            eval_ model.view.overlay.events.mouse_down ([] {
                Container::set_all_visualizations_pointer_events("auto");
            });
```